.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,47 @@
+name: Documentation
```

Comment:
@iwensu0313 this is the GitHub Action config I  "copy around" when I need to do this. The other commits are  fixes the page build, configuration, and cleanups. (This was untouched for a while and needed some updates.)